ADBench/plot_graphs.py

```diff
@@ -78,7 +84,19 @@
         t_vals = list(map(lambda pair: pair[1], times_sorted))
 
         # Plot results
-        pyplot.plot(n_vals, t_vals, label=utils.format_tool(tool), marker=marker)
+        handles += pyplot.plot(n_vals, t_vals, marker=marker, label=utils.format_tool(tool))
```

Comment:
Can we make sure thet we use different markers when we have used up colours?  Right now, I see orange+x twice on the curves, better would be orange+x, then orange++, orange+o etc.